test/test_prototype_transforms_functional.py

```diff
@@ -138,6 +138,22 @@ def make_one_hot_labels(
         yield make_one_hot_label(extra_dims_)
 
 
+def make_segmentation_mask(size=None, *, max_value=80, extra_dims=(), dtype=torch.uint8):
+    size = size or torch.randint(16, 33, (2,)).tolist()
+    shape = (*extra_dims, 1, *size)
+    data = make_tensor(shape, low=0, high=max_value, dtype=dtype)
+    return features.SegmentationMask(data)
+
+
+def make_segmentation_masks(
+    image_sizes=((32, 32), (32, 42), (38, 24)),
```

Comment:
Any reason to not use the same values as in `make_images`?

Reply:
I'd say there is no particular reason, just prefer even size images.
What was the reason for you to use `(16, 16), (7, 33), (31, 9)` ?

Reply:
OK, i'll put your numbers as I have no neither strong opinion. Just a thought, I agree that odd numbers looks like corner cases but odd sized images are less frequent to my opinion. It would make more sense then to test all combinations to ensure that the op works on square, rectangular and even/odd sized images...